### ISSUE TITLE
enum.enumbadkeyerror

### ISSUE THREAD

**Issue by Coach81** [BurntSushi/nfldb#99]
I had a lot of trouble installing nfldb.  I couldn't find pip2 and used pip to install.  The dependency files didn't install, so I installed them manually.  When I run the query in the example (passing yards) I get an error (enumeration). Perhaps I have a bad dependency file?  How can I check?  I'm very new with Linux / PostgreSQL, but I am quite familiar with standard SQL.  Oh, and I'm doing all of this on my Raspberry Pi.

**Comment by BurntSushi** [comment]
> I had a lot of trouble installing nfldb. I couldn't find pip2 and used pip to install.

Yes, the Python packaging situation sucks and varies depending on your Linux distribution.

>  The dependency files didn't install

Can you please be more specific? What command did you run? What was the output?

> When I run the query in the example (passing yards)

Can you please provide the full program that you're trying to run?

> I get an error (enumeration). 

Can you please paste the entire output? We can't guess at what's wrong. We need more information to help you.

>  Oh, and I'm doing all of this on my Raspberry Pi.

Running a PostgreSQL database on an RPi probably isn't a good idea. It would be better to run the database on a different machine on your local network and connect to it from the RPi.

**Comment by Coach81** [comment]
Hello Andrew,
Thanks for taking the time to reply.  By walking through the error messages and a bunch of Google, I was able to get the right Enum34 package to install.  Finally have everything running on my RPI.  Yup, it's slow, but I don't mind.  It was a good project for me to learn about Linux, Postgres and my Pi.
